x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/native_connector_configuration/native_connector_configuration.tsx

```diff
@@ -126,7 +128,79 @@ export const NativeConnectorConfiguration: React.FC = () => {
-          <EuiFlexGroup direction="column" />
+          <EuiFlexGroup direction="column">
+            <EuiFlexItem grow={false}>
+              <EuiPanel hasBorder hasShadow={false}>
+                <EuiFlexGroup direction="row" alignItems="center" gutterSize="xs">
+                  <EuiFlexItem grow={false}>
+                    <EuiIcon type="clock" />
+                  </EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiTitle size="xxs">
+                      <h4>
+                        {i18n.translate(
+                          'xpack.enterpriseSearch.content.indices.configurationConnector.nativeConnector.schedulingReminder.title',
+                          {
+                            defaultMessage: 'Configurable sync schedule',
+                          }
+                        )}
+                      </h4>
+                    </EuiTitle>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+                <EuiSpacer size="s" />
+                <EuiText size="s">
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.indices.configurationConnector.nativeConnector.schedulingReminder.description',
+                    {
+                      defaultMessage:
+                        'Remember to set a sync schedule in the Scheduling tab to continually refresh your searchable data.',
+                    }
+                  )}
+                </EuiText>
+              </EuiPanel>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiPanel hasBorder hasShadow={false}>
+                <EuiFlexGroup direction="row" alignItems="center" gutterSize="xs">
+                  <EuiFlexItem grow={false}>
+                    <EuiIcon type="globe" />
+                  </EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiTitle size="xxs">
+                      <h4>
+                        {i18n.translate(
+                          'xpack.enterpriseSearch.content.indices.configurationConnector.nativeConnector.securityReminder.title',
+                          {
+                            defaultMessage: 'Document level security',
+                          }
+                        )}
+                      </h4>
+                    </EuiTitle>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+                <EuiSpacer size="s" />
+                <EuiText size="s">
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.indices.configurationConnector.nativeConnector.securityReminder.description',
+                    {
+                      defaultMessage:
+                        'Restrict and personalize the read access users have to the index documents at query time.',
+                    }
+                  )}
+                  <EuiSpacer size="s" />
+                  <EuiLink href={'' /* TODO docsLink url */} target="_blank">
```

Comment:
highlight: just in case this todo isn't intentional.

Reply:
We're going back for a round of these later, we have an open issue for it!